### ISSUE TITLE
Install Update, wrong default prefix path

### ISSUE THREAD

**Issue by mysteryx93** [lutris/lutris#4983]
### Bug description

I installed Baldur's Gate 3 from GOG. When I right-click and "Install Updates", it shows me the list of updates.

When I click Install on a patch, it always sets a wrong default wine prefix path. It only says "Select installation directory" so I don't even know I have to browse to the root prefix, it's very confusing and I had to try several times before finding a path it liked.

Game prefix: /run/media/hanuman/wdc-sn730/Games/Baldurs Gate 3/

Install default prefix: /home/hanuman/Games/gog/baldurs_gate_iii

It should set the game's wine prefix by default.

Also, perhaps the message could be clearer to indicate that the "installation path" is the wine prefix root itself.

For now I have to browse to the game's path and copy/paste it every time I want to update the game.

### How to Reproduce

1. Install game
2. Right-click, "Install Updates"
3. Click "Install" on a patch

### Expected behavior

The default path should be the game's path

### Log output

```shell
Unsupported locale setting. Fix your locales

(lutris:2975352): Gtk-WARNING **: 12:21:55.981: Locale not supported by C library.
Using the fallback 'C' locale.
locale: Cannot set LC_MESSAGES to default locale: No such file or directory
locale: Cannot set LC_COLLATE to default locale: No such file or directory
INFO     2023-08-14 12:21:56,602 [startup.init_lutris:209]:Starting Lutris 0.5.13
INFO     2023-08-14 12:21:56,604 [startup.check_driver:65]:Using NVIDIA drivers 535.54.03 for x86_64
INFO     2023-08-14 12:21:56,604 [startup.check_driver:69]:GPU: NVIDIA GeForce RTX 2060
INFO     2023-08-14 12:21:56,604 [startup.check_driver:85]:GPU: 10DE:1F15 1025:1442 (nvidia drivers)
INFO     2023-08-14 12:21:56,604 [startup.check_driver:85]:GPU: 8086:9BC4 1025:1441 (i915 drivers)
DEBUG    2023-08-14 12:21:56,754 [xrandr._get_vidmodes:16]:Retrieving video modes from XrandR
DEBUG    2023-08-14 12:21:56,907 [lutris.get_missing_game_ids:191]:Checking for missing games
```

### System Information

```shell
[System]
OS:              Garuda Linux Soaring Raptor
Arch:            x86_64
Kernel:          6.4.8-x64v2-xanmod1-1
Desktop:         KDE
Display Server:  x11

[CPU]
Vendor:          GenuineIntel
Model:           Intel(R) Core(TM) i7-10750H CPU @ 2.60GHz
Physical cores:  6
Logical cores:   12

[Memory]
RAM:             15.5 GB
Swap:            27.2 GB

[Graphics]
Vendor:          NVIDIA Corporation
OpenGL Renderer: NVIDIA GeForce RTX 2060/PCIe/SSE2
OpenGL Version:  4.6.0 NVIDIA 535.54.03
OpenGL Core:     4.6.0 NVIDIA 535.54.03
OpenGL ES:       OpenGL ES 3.2 NVIDIA 535.54.03
Vulkan Version:  1.3.255
Vulkan Drivers:  Intel(R) UHD Graphics (CML GT2) (1.3.246), NVIDIA GeForce RTX 2060 (1.3.242)
```

### Checklist:

- [X] I'm not asking for support with a game or the wine runner.
- [X] I have followed the above mentioned guides and have all the graphics and wine dependencies installed.
- [X] I have checked for existing issues that describe my problem prior to opening this one.
- [X] I understand that improperly formatted bug reports may be closed without explanation.

**Comment by danieljohnson2** [comment]
This sounds strange; Lutris does not ask for the prefix location when you install an update. Does it?

Can you give us a screenshot of where it asks for this? Let's see some pixels. Pixels make everything better.

**Comment by mysteryx93** [comment]
![Screenshot_Installing Baldur's Gate 3_1](https://github.com/lutris/lutris/assets/8528033/71445c99-6baf-475f-885d-8b08df6e9384)

**Comment by danieljohnson2** [comment]
That does look like a full installation; that is not what it should it look like.

What are you doing to apply the update? Is it in the context menu?

**Comment by mysteryx93** [comment]
Right-click, context menu, "Install updates", click "install" on the patch version

**Comment by danieljohnson2** [comment]
I was going to look into this but something has come up and I don't think I can do it right now. Sorry.

Maybe I can catch up later.
